src/Algebra/Morphism/RingMonomorphism.agda

```diff
@@ -25,9 +23,14 @@ open RawRing R₁ renaming (Carrier to A; _≈_ to _≈₁_)
 open RawRing R₂ renaming
   ( Carrier to B; _≈_ to _≈₂_; _+_ to _⊕_
   ; _*_ to _⊛_; 1# to 1#₂; 0# to 0#₂; -_ to ⊝_)
-
-open import Algebra.Definitions
+import Algebra.Morphism.GroupMonomorphism  as GroupMonomorphism
+import Algebra.Morphism.MonoidMonomorphism as MonoidMonomorphism
+open import Relation.Binary.Core using (Rel)
+open import Algebra.Definitions using
+  (_DistributesOverˡ_; _DistributesOverʳ_
+  ; _DistributesOver_; LeftZero; RightZero; Zero)
```

Comment:
```suggestion
open import Algebra.Definitions
  using (_DistributesOverˡ_; _DistributesOverʳ_; _DistributesOver_
        ; LeftZero; RightZero; Zero)
```

This privileges having cognate imports on each line over possible layout consistency.